### PR TITLE
Week 1 homework Nisa

### PR DESCRIPTION
**Pull request by nisaliva**

### CODE REVIEW

File: Week1/homework/compareArrays.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+const array1 = ['Sena', 7, false, {age:7}];
+const array2 = ['Rana', 'Sena', 3, 7, true, false, 'Nisa'];
+
+console.log(array1.length);
+console.log(array2.length);
+
+if (array1 === array2) {
```
Comment (unmeshvrije) [inline Week1/homework/compareArrays.js]: please compare the length as asked in the assigment

File: Week1/homework/errorDebugging.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+console.log('I/m awesome');
```
Comment (unmeshvrije) [inline Week1/homework/errorDebugging.js]: using a forward slash is not a solution !

File: Week1/homework/logTheRemainder.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+let x = 7; // the value of x is 7.
+x = x % 3; // the new value of x is 1.
```
Comment (unmeshvrije) [inline Week1/homework/logTheRemainder.js]: Perhaps write a comment about why the value is 1

File: Week1/homework/roundAnumber.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+let z = 7.25;
+console.log(z);
+
+let a = Math.round (z);
+console.log(a);
+
+let highestOfTwo = Math.max (z,a);
```
Comment (unmeshvrije) [inline Week1/homework/roundAnumber.js]: Good job on using max() function and a long meaningful camel case name

File: Week1/homework/compareArrays.js (hunk 1 above)
Comment (nisaliva) [reply Week1/homework/compareArrays.js]: Ok. It should be like this ::
if (array1.length === array2.length)

File: Week1/homework/errorDebugging.js (hunk 2 above)
Comment (nisaliva) [reply Week1/homework/errorDebugging.js]: yes. I think the correct solution is with "\" .

File: Week1/homework/logTheRemainder.js (hunk 3 above)
Comment (nisaliva) [reply Week1/homework/logTheRemainder.js]: Because the remainder for 7%3 is 1.

File: Week1/homework/roundAnumber.js (hunk 4 above)
Comment (nisaliva) [reply Week1/homework/roundAnumber.js]: That's nice to know. Thank you:)

**Comment by nisaliva** [comment]
Thank you @unmeshvrije  for your feedback!

**Pull request closed without merging**